### ISSUE TITLE
[bug]: Error when trying to close right side window

### ISSUE THREAD

**Issue by ClaimedLizard** [shortcuts/no-neck-pain.nvim#407]
### Description

An error message occurs when trying to close a window. Upon pressing Enter to dismiss the error, the closed window immediately comes back. 

Specifically, this error only occurs when the blank side buffers are currently hidden because their width is under the configured `minSideBufferWidth`, and closing the right-most window is enough to bring the blank side buffers back.
Closing the left-most window, however, does not produce the error.

See the reproduction steps for screenshots.

The exact error message is as follows:
```
Error executing vim.schedule lua callback: ...re/nvim/lazy/no-neck-pain.nvim/lua/no-neck-pain/main.lua:125: Invalid window id: 1007                                                                    
stack traceback:                                                                                                                                                                                               
        [C]: in function 'nvim_set_current_win'                                                                                                                                                                
        ...re/nvim/lazy/no-neck-pain.nvim/lua/no-neck-pain/main.lua:125: in function 'callback'                                                                                                                
        ...vim/lazy/no-neck-pain.nvim/lua/no-neck-pain/util/api.lua:115: in function <...vim/lazy/no-neck-pain.nvim/lua/no-neck-pain/util/api.lua:113>
```

And here is my no-neck-pain configuration:
```
    { 'shortcuts/no-neck-pain.nvim',
        config = function()
            require("no-neck-pain").setup({
                width = 100,
                minSideBufferWidth = 5,

                autocmds = {
                    enableOnVimEnter = true,
                    enableOnTabEnter = true,
                    reloadOnColorSchemeChange = true,
                    skipEnteringNoNeckPainBuffer = false,
                },

                mappings = {
                    enabled = true,
                    toggle = "<Leader><Leader>",
                    toggleLeftSide = "false",
                    toggleRightSide = "false",
                    widthUp = "false",
                    widthDown = "false",
                    scratchPad = "false",}
            })
        end
    }
```

I also experience the issue on my other computer, which has the same configuration, but with a higher value for `minSideBufferWidth` configured.

Interestingly, this issue is not present on version `1.14`, so I am pinning this plugin to the older version for now.

Anyway, thanks for the great work with this plugin! It has definitely saved my neck from a lot of pain over the months that I have been using it lol.

### Neovim version

0.9.x

### Steps to reproduce

1. Configure `minSideBufferWidth such that, when there is only one window open, the blank side buffers appear, and on opening another vertical split, the blank side buffers disappear.
![screenshot_1](https://github.com/user-attachments/assets/f10eab38-acc8-4b0a-8927-7f22524847b4)
![screenshot_2](https://github.com/user-attachments/assets/72c445de-cf44-4af7-aba8-f2b0caa90f37)

2. Close the window on the right side with Ctrl+w c.
The error message appears.
![screenshot_3](https://github.com/user-attachments/assets/aafe98a5-0403-43f1-991a-d8e66a014b92)

3. Pressing Enter to dismiss the error message causes the just-closed window to come back.
![screenshot_4](https://github.com/user-attachments/assets/acf4b283-abc3-488c-a8d4-179cd1f2f75f)

### Relevant log output

_No response_

### Self-service

- [ ] I'd be willing to fix this bug myself.

**Comment by shortcuts** [comment]
Hey there, thanks for using nnp and reporting the issue!

Indeed we should do a check before this call 😮‍💨 good catch! I've not properly been able to reproduce your error, but found a subsequent one that caused what you describe in step 3

Should be fixed with https://github.com/shortcuts/no-neck-pain.nvim/pull/409

**Comment by shortcuts** [comment]
fix released in 2.0.6
